frontend/src/tests/utils/sortHelper.test.js

```diff
@@ -0,0 +1,36 @@
+import { compareValues } from "main/utils/sortHelper"
+
+describe("sortHelper tests", () => {
+  
+   var singers = [];
+
+   const tyler = { name: 'Steven Tyler', band: 'Aerosmith', born: 1948 };
+   const carpenter = { name: 'Karen Carpenter', band: 'The Carpenters', born: 1950 };
+   const cobain = { name: 'Kurt Cobain', band: 'Nirvana', born: 1967 };
+   const nicks = { name: 'Stevie Nicks', band: 'Fleetwood Mac', born: 1948 };
+
+  beforeEach(() => {
+    singers = [ tyler, carpenter, cobain, nicks ]
+  });
+
```

Comment:
This code, if I recall correctly, was needed for tests for the table format that we used to use, but no longer do. 

Can you point to where this code is needed in the new code base?  if so, then we can keep it... but I'm suspecting that it might not be needed at all now that we have pivoted to using React Table Library instead react-bootstrap-table-next